library/sequencer/src/lib/ui/sequence.rb

```diff
@@ -6,6 +6,73 @@ module UI
   # {Yast::SequencerClass Yast::Sequencer}.
   # In the simple case it runs a sequence of dialogs
   # connected by Back and Next buttons.
+  #
+  # @example simple two steps sequence
+  #   class SimpleSequence < UI::Sequence
+  #     SEQUENCE_HASH = {
+  #       START   => "step1",
+  #       "step1" => {
+  #         next: "step2",
+  #       },
+  #       "step2" => {
+  #         next: :finish
+  #       }
+  #     }
+  #
+  #     def step1
+  #       :next
```

Comment:
What about adding `# ... show UI of this step ...` above the return value?

Reply:
well, when I thinking about this it does not need to be UI. In fact it is just some kind of StateMachine with defined transitions between states.